mz_os.h

```diff
@@ -21,7 +21,7 @@ extern "C" {
 #  define MZ_VERSION_MADEBY_HOST_SYSTEM (MZ_HOST_SYSTEM_OSX_DARWIN)
 #elif defined(__riscos__)
 #  define MZ_VERSION_MADEBY_HOST_SYSTEM (MZ_HOST_SYSTEM_RISCOS)
-#elif defined(__unix__)
+#elif defined(__unix__) || defined(__midipix__)
```

Comment:
This should be moved below `_WIN32`  and only use `#else`.

Reply:
Fair enough, fixed:-)